themes/default/css/_bundleLabel.scss

```diff
@@ -1,37 +1,55 @@
-
-
 .bundleLabel {
-	font-size: inherit;
-	font-family: inherit;
-	color: inherit;
-	background-color: white;
-	border: 1px solid white;
-	box-shadow: 0 0 3px 1px rgb(220,220,220);
-
-  border-radius: 3px;
-  margin-bottom: 1em;
+	background-color: transparent;
+	border: none;
+	padding: 0;
+	margin: 0;
+
+	.bundleInner {
+		border-top: 1px solid $white-off;
+		padding: 8px;
+		margin: 8px 0 0 0;
+	}
 
+	input[type="text"],
+	input[type="password"],
+	input[type="textarea"],
+	textarea,
+	select {
+		width: 100% !important;
+	}
 }
 
-
+// As a fallback, make children that don't have a `cell-*` class like a 100% wide cell.  See `_grid.scss`
+.grid > .bundleLabel:not([class *= 'cell-']) {
+	display: inline-block;
+	vertical-align: top;
+	font-size: medium;
+	width: 100%;
+}
 
 .bundleContainer {
 	background: inherit;
 }
 
+.bundleShowHideControl {
```

Comment:
![hideoverlapsdelete](https://cloud.githubusercontent.com/assets/12571/11234635/da1e824c-8e04-11e5-8a01-247f092124f1.png)
The hide button is overlapped by the delete item button, and also it's pretty hard to click on the collapse button.